dependencies.yaml

```diff
@@ -35,6 +36,30 @@ dependencies:
           - cxx-compiler
           - make
     specific:
+      - output_types: conda
+        matrices:
+          - matrix:
+              cuda: "11.[245]"
+            packages:
+              - gcc<11.0.0
+          - matrix:
+              cuda: "11.[68]"
+            packages:
+              - gcc<12.0.0
+          - matrix:
+              cuda: "12.*"
+            packages:
+              - gcc<13.0.0
+      - output_types: conda
+        matrices:
+          - matrix:
+              arch: x86_64
+            packages:
+              - sysroot_linux-64==2.17
```

Comment:
It looks like we were requiring `sysroot_linux-64` on ARM builds before this PR. I fixed that.